submissions/expero1/html-css-popup/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./style.css">
+	<title>Document</title>
+	
+</head>
+<body>
+	<header>
+		<div class='header-wrapper'>
+			<div class="logo">
+				<img src='img/favicon.png' alt='Kottans'>
+			</div>
```

Comment:
The logo should be a link and lead to the home page.

Reply:
Thank You. That's my fault. I was in a hurry and was not attentive. Fixed.